.github/workflows/sdk_publish.yaml

```diff
@@ -23,3 +23,4 @@ jobs:
       java_gpg_secret_key: ${{ secrets.JAVA_GPG_SECRET_KEY }}
       ossrh_password: ${{ secrets.OSSRH_PASSWORD }}
       speakeasy_api_key: ${{ secrets.SPEAKEASY_API_KEY }}
+      SHIPPO_TOKEN: ${{ secrets.SHIPPO_TOKEN }}
```

Comment:
This will fix failing tests (here)[https://github.com/goshippo/shippo-java-sdk/actions/runs/11959497869/job/33341468175]

Proved locally by unsetting env var and replicating error.